framework/src/modules/chain/chain.js

```diff
@@ -174,24 +173,27 @@ module.exports = class Chain {
 
 			self.logger.info('Modules ready and launched');
 
-			this.channel.subscribe(
-				'network:subscribe',
-				({ data: { event, data } }) => {
-					if (event === 'postTransactions') {
-						this.scope.modules.transport.shared.postTransactions(data);
-						return;
+			// Avoid receiving blocks/transactions from the network during snapshotting process
+			if (!this.options.loading.snapshotRound) {
+				this.channel.subscribe(
+					'network:subscribe',
+					({ data: { event, data } }) => {
+						if (event === 'postTransactions') {
```

Comment:
Is this check correct? 
You are checking twice for the same event name here (line `181` and `185`)

Reply:
Good catch!

@jondubois line `185` should be `postSignatures`, right?

Reply:
must be merge mistake... =( it should be postSignatures